schemas/mozilla_nimbus_schemas/tests/jetstream/test_population_sizing.py

```diff
@@ -1,7 +1,7 @@
 import json
 from unittest import TestCase
 
-from mozilla_nimbus_schemas.jetstream import SampleSizes
+from mozilla_nimbus_schemas.jetstream import SampleSizes, SampleSizesFactory
```

Comment:
I don't see where this is used in the tests?

Reply:
Sorry thought I removed that. I added it to make sure it worked, but for the package I thought I'd keep using the already-written static test data.

Reply:
Oh, if the factories aren't used in the tests then there's no way to assert that they're working?